website/build.gradle

```diff
@@ -83,6 +83,18 @@ task startDockerContainer(type: Exec) {
     "${->createDockerContainer.containerId()}" // Lazily evaluate containerId.
 }
 
+task setupDockerContainer(type: Exec) {
+  dependsOn startDockerContainer
+  ext.containerId = {
+    return startDockerContainer.containerId()
+  }
+
+  // Create the config to point to a GitHub or Colab blob in the repo, e.g. apache/beam/blob/master
```

Comment:
Since this temporary file would always be created, do we need to the branch_repo config definition in _config.yml.

Reply:
The temporary file is always created but it has a different configuration depending on the branch you're on, so it will have different values on every pull request. That's why it's created dynamically.

Note that even though it's always created, it's only used on `buildLocalWebsite` and `buildGcsWebsite`, for testing and staging. But it won't be used on `buildApacheWebsite`, so it will point to the `apache/beam/blob/master` defined in the `_config.yml`.